web-common/src/features/dashboards/time-controls/TimeGrainSelector.svelte

```diff
@@ -2,17 +2,21 @@
   import CaretDownIcon from "@rilldata/web-common/components/icons/CaretDownIcon.svelte";
   import WithSelectMenu from "@rilldata/web-common/components/menu/wrappers/WithSelectMenu.svelte";
   import { createEventDispatcher } from "svelte";
+  import { useDashboardStore } from "../dashboard-stores";
   import type { TimeGrain } from "./time-control-types";
   import { prettyTimeGrain, TimeGrainOption } from "./time-range-utils";
 
-  export let selectedTimeGrain: TimeGrain;
-  export let selectableTimeGrains: TimeGrainOption[];
+  export let metricViewName: string;
```

Comment:
It's better to revert this to it's original prop. With default and minimum time grain selectors coming in, the handling logic for selected time grain would be tightly coupled with the time ranges.

Reply:
There were a couple changes here:
1. Changing terminology: `selectedTimeGrain` -> `activeTimeGrain`, `selectableTimeGrains` -> `timeGrainOptions`
2. Reading the `activeTimeGrain` from the `dashboardStore`, rather than feeding it in as a prop

We're still feeding the `selectableTimeGrains`/`timeGrainOptions` into the component. So I think we're good?

Reply:
Yes, we can leave this one as is. The default selector work also touches some of these components so I will handle it as needed.